app/helpers/peer_review_helper.rb

```diff
@@ -45,10 +43,13 @@ def create_map_group_id_to_name(reviewer_groups, reviewee_groups)
   # Returns a map of reviewer_id => num_of_reviews
   def create_map_number_of_reviews_for_reviewer(reviewer_groups)
     number_of_reviews_for_reviewer = {}
+
+    # TODO - [N+1 query] Optimize with: SELECT reviewer_id, COUNT(reviewer_id) FROM peer_reviews GROUP BY reviewer_id
```

Comment:
Do something like `number_of_reviews_for_reviewer = PeerReview.group(:reviewer_id).having(reviewer_id: reviewer_groups.map { |g| g['id'] }).count` (note that `count` returns a hash, I believe).

I've lost track of the type of `reviewer_groups`. If it's a plain dictionary you have to do `['id']`, otherwise you might be able to do `&:id` if it's an array of objects.